src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -2622,7 +2629,7 @@ protected function getArrayableItems(array $values)
      */
     public function getAttribute($key)
     {
-        if (array_key_exists($key, $this->attributes) || $this->hasGetMutator($key)) {
+        if (array_key_exists($key, $this->attributes) || $this->hasGetMutator($key) || $this->isAlias($key)) {
```

Comment:
No, I suppose I didn't end up using it.  It could be used in a user function I guess.  Should I remove it?